v2/internal/testcommon/test_context.go

```diff
@@ -306,14 +306,20 @@ func createRecorder(cassetteName string, cfg config.Values, recordReplay bool) (
 			}
 		}
 
-		for _, values := range i.Response.Headers {
+		for key, values := range i.Response.Headers {
 			for i := range values {
 				values[i] = hide(values[i], azureIDs.subscriptionID, uuid.Nil.String())
 				values[i] = hide(values[i], azureIDs.tenantID, uuid.Nil.String())
 				if azureIDs.billingInvoiceID != "" {
 					values[i] = hide(values[i], azureIDs.billingInvoiceID, DummyBillingId)
 				}
 			}
+			// Hide the base request URL in the AzureOperation and Location headers
+			if key == "Azure-Asyncoperation" || key == "Location" {
```

Comment:
Do we have any constants for these response headers somewhere? Can these response header values change in future?

Reply:
Tryna use them below? 

https://github.com/Azure/azure-sdk-for-go/blob/sdk/armcore/v0.8.1/sdk/armcore/internal/pollers/pollers.go#L22

Reply:
Those ones are internal so we cant use them, but Ill define our own and use them